examples/src/test/java/com/azure/cosmos/cassandra/CosmosCassandraExtensionsExample.java

```diff
@@ -0,0 +1,262 @@
+ * this name already exist, it is reused.
+ * <li>Creates a table within the keyspace created or reused. If a table with the given name already exists, it is
+ * reused.
+ * <li>The keyspace created or reused is then dropped. This prevents keyspaces from accumulating with repeated test
+ * runs.</ol>
+ * <h3>
+ * Notes</h3>
+ * <ul>
+ * <li>You should never attempt to retry a non-idempotent write. See the driver's manual page on idempotence for more
+ * information.</ul>
+ *
+ * @see CosmosLoadBalancingPolicy
+ * @see CosmosRetryPolicy
+ * @see <a href="../../../../doc-files/application.conf.html">application.conf</a>
+ * @see <a href="../../../../doc-files/reference.conf.html">reference.conf</a>
+ * @see <a href="https://docs.datastax.com/en/developer/java-driver/4.9/manual/">DataStax Java Driver manual</a>
+ * @see <a href="https://docs.datastax.com/en/developer/java-driver/4.9/manual/core/configuration/">DataStax Java Driver
+ * configuration</a>
+ */
+public class CosmosCassandraExtensionsExample {
+
+    // region Fields
+
+    private static final ConsistencyLevel CONSISTENCY_LEVEL = ConsistencyLevel.QUORUM;
+    private static final int TIMEOUT = 30_000;
+
+    private CqlSession session;
+
+    // endregion
+
+    // region Methods
+
+    /**
+     * Shows how to integrate with a Cosmos Cassandra API instance using azure-cosmos-cassandra-driver-4-extensions.
+     */
+    @Test(groups = { "examples" }, timeOut = TIMEOUT)
+    public void canIntegrateWithCosmos() {
+
+        try (final Session ignored = this.connect()) {
+
+            assertThatCode(this::createSchema).doesNotThrowAnyException();
+
+            assertThatCode(() ->
+                this.write(CONSISTENCY_LEVEL)
+            ).doesNotThrowAnyException();
+
+            assertThatCode(() -> {
+                final ResultSet rows = this.read(CONSISTENCY_LEVEL);
+                this.display(rows);
+            }).doesNotThrowAnyException();
+
+        } catch (final Exception error) {
+            final StringWriter stringWriter = new StringWriter();
+            error.printStackTrace(new PrintWriter(stringWriter));
+            fail(format("connect failed with %s", stringWriter));
+        }
+    }
+
+    // region Privates
+
+    /**
+     * Initiates a connection to the cluster specified by application.conf.
+     */
+    private Session connect() {
+        this.session = CqlSession.builder().build();
+        System.out.println("Connected to session: " + this.session.getName());
+        return this.session;
+    }
+
+    /**
+     * Creates the schema (keyspace) and table to verify that we can integrate with Cosmos.
+     */
+    private void createSchema() {
+
+        this.session.execute(SimpleStatement.newInstance(
+            "CREATE KEYSPACE IF NOT EXISTS downgrading WITH replication = {"
+                + "'class':'SimpleStrategy',"
+                + "'replication_factor':3"
```

Comment:
internally we ignore this but would be good to keep this 4 to clarify our internal replication factor.